feat(changelog): webhosting-removed-plesk-option-removed 2025-05-12 (#4939)
scaleway/docs-content: changelog/may2025/2025-05-12-webhosting-removed-plesk-option-removed.mdx

---

> title: Plesk Option Removed
> status: removed
> date: 2025-05-12
> category: managed-services
> product: webhosting
> ---
> 
> To streamline our Web Hosting offer, the Plesk option has been removed for new creations.
> 
> Existing Web Hosting with Plesk remain fully functional and supported. 
> However, we encourage users to explore alternative solutions available within our platform for new deployments.
> 

Reviewer: ```suggestion
title: Plesk option removed
status: removed
date: 2025-05-12
category: managed-services
product: webhosting
---

To streamline our Web Hosting offer, the Plesk option has been removed for new creations. Existing web hostings with Plesk remain fully functional and supported. However, we encourage users to explore alternative solutions available in the [Scaleway console](https://console.scaleway.com/webhosting) for new deployments.
```
